packages/ember-htmlbars/lib/helpers/if_unless.js

```diff
@@ -64,6 +64,7 @@ function appendBlockConditional(view, inverted, helperName, params, hash, option
     view.appendChild(BoundIfView, {
       _morph: options.morph,
       _context: get(view, 'context'),
+      renderer: get(view, 'renderer'),
```

Comment:
I wonder why we need to thread it through like this also. I thought perhaps it could be added at a higher level (to prevent having to sprinkle the wiring through everywhere). Possibly like adding to 'appendChild'?